main.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
One nit about being importable. When I started someone told me the pattern was to do

```
- cmd
  - binaryname
    - main.go
- internal
  - foo
    - foo.go
  - bar
    - bar.go
```

which works pretty well. It's very easy to move stuff out and make it possible to depend on them. Very hard to go the other direction.

using the `cmd/binaryname` is useful because when you `go build github.com/vsinha/launchdata/cmd/{cacher,runner,foo}@latest` you'll get a binary called `cacher`, `runner`, and `foo`.